packages/frontend/src/components/UserSettings/UserAttributesPanel/UserAttributeModal.tsx

```diff
@@ -0,0 +1,201 @@
+                                        name={`users.${index}.userUuid`}
+                                        placeholder="E.g. test@lightdash.com"
+                                        required
+                                        searchable
+                                        {...form.getInputProps(
+                                            `users.${index}.userUuid`,
+                                        )}
+                                        data={
+                                            orgUsers?.map((orgUser) => ({
+                                                value: orgUser.userUuid,
+                                                label: orgUser.email,
+                                            })) || []
+                                        }
+                                    />
+
+                                    <TextInput
+                                        name={`users.${index}.value`}
+                                        placeholder="E.g. US"
+                                        required
+                                        {...form.getInputProps(
+                                            `users.${index}.value`,
+                                        )}
+                                    />
+                                    <Button
+                                        pr={5}
+                                        leftIcon={
+                                            <MantineIcon icon={IconCircleX} />
+                                        }
+                                        color="red"
+                                        variant="outline"
+                                        onClick={() => {
+                                            form.setFieldValue(
+                                                'users',
+                                                form.values.users.filter(
+                                                    (_, i) => i !== index,
+                                                ),
+                                            );
+                                        }}
+                                    />
+                                </Group>
+                            );
+                        })}
+                        <Button
+                            w={200}
+                            leftIcon={<MantineIcon icon={IconPlus} />}
+                            onClick={() => {
+                                form.setFieldValue('users', [
+                                    ...(form.values.users || []),
+                                    { userUuid: '', email: '', value: '' },
+                                ]);
+                            }}
+                        >
+                            Add user
+                        </Button>
+                    </Stack>
+
+                    <Group spacing="xs" position="right" mt="md">
```

Comment:
if you have a parent stack of this set to spacing='lg' you don't really need to use `mt` anymore